core/mutator.js

```diff
@@ -385,9 +385,9 @@ Blockly.Mutator.prototype.workspaceChanged_ = function() {
 Blockly.Mutator.prototype.getFlyoutMetrics_ = function() {
   return {
     viewHeight: this.workspaceHeight_,
-    viewWidth: this.workspaceWidth_,
+    viewWidth: this.workspaceWidth_ - this.workspace_.flyout_.width_,
```

Comment:
Use `this.workspace_.getFlyout_().getWidth()`.

The goal is to not directly access private properties, and to use setters and getters instead, even if that makes it more verbose.  That way if we change something internally (say, `getWidth()` changes to add a constant value to the stored `width_`) we don't need to find every single place that directly accesses that property.

`getFlyout_` on workspaceSvg is `package`, even though it has an underscore in the name (historical problem).

Accessing `this.workspace_` is fine because it's a private property on the object we're currently on.

There's plenty of code that doesn't follow these rules, but we try to make sure new code does, and that we update old code as we touch it.